CODEOWNERS

```diff
@@ -246,6 +246,7 @@ Makefile* @cilium/build
 /bpf/Makefile* @cilium/loader
 /bpf/init.sh @cilium/loader
 /bpf/custom/Makefile* @cilium/build @cilium/loader
+/bpf/lib/auth.h @cilium/sig-servicemesh
```

Comment:
Should this have @cilium/sig-datapath _and_ @cilium/sig-servicemesh ?

I'd like to see an ack from datapath side before merging.

Reply:
definitely agree. it was meant to stay up to date if something changes. -> included sig-datapath